packages/api/tests/endpoints/executions.js

```diff
@@ -1,234 +0,0 @@
-const { app } = require('../../app');
-
-test.before(async (t) => {
-  const esAlias = randomString();
-  process.env.ES_INDEX = esAlias;
-  await bootstrapElasticSearch('fakehost', esIndex, esAlias);
-
-  await awsServices.s3().createBucket({ Bucket: process.env.system_bucket }).promise();
-
-  const username = randomString();
-  await setAuthorizedOAuthUsers([username]);
-
-  accessTokenModel = new AccessToken();
-  await accessTokenModel.createTable();
-
-  jwtAuthToken = await createFakeJwtAuthToken({ accessTokenModel, username });
-
-  esClient = await Search.es('fakehost');
-
-  process.env = {
-    ...process.env,
-    ...localStackConnectionEnv,
-    PG_DATABASE: testDbName,
-  };
-
-  const { knex, knexAdmin } = await generateLocalTestDb(testDbName, migrationDir);
-  t.context.testKnex = knex;
-  t.context.testKnexAdmin = knexAdmin;
-
-  t.context.executions = [
-    {
-      arn: 'arn1',
-      status: 'running',
-      name: 'test_execution',
-    },
-    {
-      arn: 'arn2',
-      status: 'completed',
-      asyncOperationId: '0fe6317a-233c-4f19-a551-f0f76071402f',
-      name: 'test_execution',
-    },
-  ];
-
-  const executionIndexPromises = t.context.executions
-    .map((execution) => indexer.indexExecution(esClient, execution, esAlias));
-
-  await Promise.all(executionIndexPromises);
-
-  await esClient.indices.refresh();
-
-  t.context.executionDynamoModel = new Execution();
-  await t.context.executionDynamoModel.createTable();
-
-  t.context.executionPgModel = new ExecutionPgModel();
-});
-
-test.after.always(async (t) => {
-  await accessTokenModel.deleteTable();
-  await recursivelyDeleteS3Bucket(process.env.system_bucket);
-  await esClient.indices.delete({ index: esIndex });
-
-  await destroyLocalTestDb({
-    knex: t.context.testKnex,
-    knexAdmin: t.context.testKnexAdmin,
-    testDbName,
-  });
-});
-
-test('GET without pathParameters and without an Authorization header returns an Authorization Missing response', async (t) => {
-  const response = await request(app)
-    .get('/executions')
-    .set('Accept', 'application/json')
-    .expect(401);
-
-  assertions.isAuthorizationMissingResponse(t, response);
-});
-
-test('GET with pathParameters and without an Authorization header returns an Authorization Missing response', async (t) => {
-  const response = await request(app)
-    .get('/executions/asdf')
-    .set('Accept', 'application/json')
-    .expect(401);
-
-  assertions.isAuthorizationMissingResponse(t, response);
-});
-
-test('GET without pathParameters and an invalid access token returns an unauthorized response', async (t) => {
-  const response = await request(app)
-    .get('/logs/executions')
-    .set('Accept', 'application/json')
-    .set('Authorization', 'Bearer ThisIsAnInvalidAuthorizationToken')
-    .expect(401);
-
-  assertions.isInvalidAccessTokenResponse(t, response);
-});
-
-test('GET logs returns all executions', async (t) => {
```

Comment:
Did this test get preserved in the move?

Reply:
Not specifically, but it seemed this was covered within the test `default returns list of executions`

Reply:
Yeah this test looks unnecessary. Thanks